src/components/layout/global-style.ts

```diff
@@ -21,6 +21,16 @@ export const GlobalStyle = createGlobalStyle<{ $lightTheme?: boolean }>`
     -webkit-font-smoothing: antialiased;
     -moz-osx-font-smoothing: grayscale;
   }
+
+  button, select {
+    font-family: ${(props) => props.theme.typography.fontFamily},
+    "Helvetica",
+    "Lucida Grande",
+    Tahoma,
+    Sans-Serif;
+    -webkit-font-smoothing: antialiased;
+    -moz-osx-font-smoothing: grayscale;
+  }
```

Comment:
I suggest to include `input` and `textarea` here as well. Also, instead of duplicating the font definition it's enough to specify `font-family: inherit;`. The font-smoothing properties are already inherited.
```suggestion
  button, select, input, textarea {
    font-family: inherit;
  }
```

Reply:
@fischeversenker Thanks for your review 🙏 It looks much clearer now